tuxemon/item/conditions/__init__.py

```diff
@@ -1,26 +0,0 @@
-#
```

Comment:
Sorry, I would rather you leave these in for now, until I have confirmation that it's ok to remove them.

If you are creating a file from scratch then feel free to leave it out.

Reply:
No problem, I'm going to reverse.